jetbrains-ultimate/build.gradle.kts

```diff
@@ -18,12 +18,12 @@ dependencies {
 val ideProfile = IdeVersions.ideProfile(project)
 
 intellij {
-    val parentIntellijTask = rootProject.intellij
+    pluginName = "aws-toolkit-jetbrains"
+
     version = ideProfile.ultimate.sdkVersion
     setPlugins(*ideProfile.ultimate.plugins)
-    pluginName = parentIntellijTask.pluginName
-    updateSinceUntilBuild = parentIntellijTask.updateSinceUntilBuild
-    downloadSources = parentIntellijTask.downloadSources
+
+    downloadSources = false
```

Comment:
Add the same comment from rider?

Reply:
Re-wrote the comments, im not sure the RD one is even correct anymore (not that it matters)